torchci/components/JobConclusion.tsx

```diff
@@ -15,7 +17,7 @@ export default function JobConclusion({
           classified ? styles["classified"] : styles[conclusion ?? "none"]
```

Comment:
I think it's worth the effort to tweak the CSS a bit here (and in https://github.com/pytorch/test-infra/blob/main/torchci/components/JobConclusion.module.css) to show a slightly different color for successful retry.  It's indeed difficult to see because the number of successful retry (F) is very few. I could only spot one in the whole page of https://torchci-git-csl-flakyhud-fbopensource.vercel.app/hud/pytorch/pytorch/master/1?per_page=50, it's like finding Waldo :)

Reply:
Can we get the grouped view to also show an `F` if any of the jobs inside it are flaky?

That would help a lot with visibility, and highlight that there's something worth knowing within that group's jobs

Reply:
changed the color to be "lightgreen", not a huge fan tho so if people have better ideas i am very open
added stuff for groups